doc/user/bgp.rst

```diff
@@ -2665,6 +2665,16 @@ structure is extended with :clicmd:`show bgp [afi] [safi]`.
 
    Display flap statistics of routes of the selected afi and safi selected.
 
+.. index:: show bgp [afi] [safi] statistics
+.. clicmd:: show bgp [afi] [safi] statistics
+
+   Display statistics of routes of the selected afi and safi.
+
+.. index:: show bgp statistics-all
+.. clicmd:: show bgp statistics-all
```

Comment:
Leaving this comment here, since the code is a bit gnarly.

Is there a reason we need `statistics-all`, when `[afi] [safi]` is already optional in the previous command?
What happens when you omit `[afi] [safi]` in the previous command? I would expect that to show stats for all afi/safi combinations.

Reply:
if you dont omit afi and safi, it will fallback to afi = ipv6 and safi = unicast.
this is default behavior.
so I can not change behavior of this.